e2e-tests/upgrade-consistency/conf/some-name-rs0.yml

```diff
@@ -1,9 +1,10 @@
-apiVersion: psmdb.percona.com/v1
+apiVersion: psmdb.percona.com/v1-10-0
 kind: PerconaServerMongoDB
 metadata:
   name: some-name
 spec:
   #platform: openshift
+  crVersion: 1.10.0
```

Comment:
Do we need crVersion here? What has changed?

Reply:
@hors It was removed at my questioning, but maybe actually we should have it since v1.10.0 supports it, previous versions did not have crVersion so we used API only?